OctaneVSPlugin/MainWindowPackage.cs

```diff
@@ -95,7 +96,7 @@ protected override void Initialize()
 
         internal void CreateFile(string fileName, string content)
         {
-            DTE dte = (DTE)GetService(typeof(DTE));
+            DTE2 dte = (DTE2) GetService(typeof(DTE));
```

Comment:
Why was this change needed? And why pass typeof(DTE) instead of typeof(DTE2) ?

Reply:
In .net 4.7.2 DTE became the child of DTE2 - so you need to cast it to DTE2 in order to have access to functions and methods from that interface